rocks/RocksDatabase.java

```diff
@@ -143,14 +144,15 @@ RocksSession createAndOpenSession(Arguments.Session.Type type, Options.Session o
         return session;
     }
 
-    synchronized Cache cache() {
+    synchronized Cache borrowCache() {
```

Comment:
When an object borrows the cache, it calls `cache::borrow` to increment the number of borrower before returning the cache object